src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+import { CarouselProvider, Slider, Slide, DotGroup } from "pure-react-carousel";
+import React from "react";
+
+import "pure-react-carousel/dist/react-carousel.es.css";
+import s from "./BlockQuoteBlock.module.css";
+
+const CarouselPropType = PropTypes.shape({
+  quote: PropTypes.string.isRequired,
+  attribution: PropTypes.string.isRequired,
+});
+
+const SlideQuote = ({ quote, attribution }) => (
+  <figure className={s.slideQuote}>
+    <blockquote className={s.quote}>&ldquo;{quote}&rdquo;</blockquote>
+    <figcaption className={s.attribution}>&mdash;{attribution}</figcaption>
+  </figure>
+);
+
+SlideQuote.propTypes = {
+  quote: PropTypes.string.isRequired,
+  attribution: PropTypes.string.isRequired,
+};
+
+const BlockQuoteBlock = ({ quotes }) => (
+  <div className={s.bleedWrapper}>
+    <div className={s.carouselWrapper}>
+      <CarouselProvider totalSlides={quotes.length} isIntrinsicHeight>
```

Comment:
Do you think you could configure this to autoplay? I think it'd look nice like that, and it'd help reinforce to the user that this is a carousel.

It looks like setting `isPlaying` will enable that. I played with it locally, and it also looks like the default `interval` is too short (5000 milliseconds), so maybe we could also set that to 10000 milliseconds?

Reply:
👍 Cool addition! Added!